content/getting-started/connect-the-pim-4x/step-2.md

```diff
@@ -0,0 +1,120 @@
+
+Couldn't find any connector that suits you?  
+It is the case for our example in this tutorial. We want to connect the PIM to our ERP but the marketplace doesn't provide any connector that covers this use case.  
+Alright! So let's build it ourselves!
+
+## Build your very own connector
+
+In the case you can't find an already cooked connector on our marketplace, you will have to develop it. 
+
+Below are the macro steps to successfully go through this development.
+1. Have your connection use case in mind
+2. Find out what are the third-party connection possibilities
+3. Choose your integration frequency
+4. Explore our API reference
+5. Think about your architecture
+6. Ochestrate your global connector
+
+Those steps are detailed below, illustrated by our tutotrial example.
+
+### 1- Have your connection use case in mind
+
+We know it can seem a bit obvious but it really is the first step for a successful connection between the PIM and your third-party. You need to have at least the global scenario of your connection in mind.
+
+#### In this tutorial
+_You want to connect your ERP. Each day, new products are created into your ERP. You want those new products to be sent over to the PIM so that you can enrich them, in the PIM, the day after._
+
+### 2- Find out what are the third-party connection possibilities
+
+Take a closer look at the software you want to connect to. How can you extract the data you will need? Does it have an API? Does it allow exporting flat files?  
+
+#### In this tutorial
+_In our example, let's say that you took a look at your ERP and discovered that, like many ERPs, it is a bit old-school. It only provides exports of products in XML files._
+
+### 3- Choose your integration frequency
+Define at which frequency you want your connector to integrate data into the PIM or retrieve it, if it is a destination. You need to choose this frequency carefully as it can have an impact on the PIM load.
+
+#### In this tutorial
+_In our example, as stated above, we know that every day, new products are created into the ERP. So we define that our connector will integrate new products, once a day at midnight, outside working hour, to lower as much as possible the PIM load._
+
+### 4- Explore our API reference
+In this step, you will need to find in our [API reference](api-reference-index.html) the endpoints you will need to achieve such a connection. 
+
+#### In this tutorial
+_For our example, you will need to create and update products in the PIM so we will mainly need this endpoint: [Patch](/api-reference.html#patch_products) on products._
+
+### 5- Think about your architecture
+Choose the architecture you want for your connector. Indeed, you will have two options here:
+    - You have access to the third-party code source and can develop your connector directly in it.
+    - You can otherwise create a middleware that will sit right in the middle of the tird-party you want to connect and the PIM.
```

Comment:
```suggestion
    - You can otherwise create a middleware that will sit right in the middle of the third-party you want to connect, and the PIM.
```